src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -80,11 +109,10 @@ s32 EnHoll_IsKokiriLayer8(void) {
 }
 
 void EnHoll_ChooseAction(EnHoll* this) {
-    s32 action;
+    s32 action = ENHOLL_GET_TYPE(&this->actor);
```

Comment:
```suggestion
    s32 type = ENHOLL_GET_TYPE(&this->actor);
```
type seems to make more sense now.